e2e-polybft/e2e/network_test.go

```diff
@@ -20,7 +20,7 @@ func TestE2E_NetworkDiscoveryProtocol(t *testing.T) {
 	)
 
 	// create cluster
-	cluster := framework.NewTestCluster(t, 10,
+	cluster := framework.NewTestCluster(t, validatorCount+nonValidatorCount,
```

Comment:
Why are you suming `validatorCount` and `nonValidatorCount`?

```suggestion
	cluster := framework.NewTestCluster(t, validatorCount,
```

Reply:
Mistake, good catch. Fix: https://github.com/0xPolygon/polygon-edge/pull/1378/commits/e796176e70d7865daae6f3b1e410e8856678e398